### DISCUSSION TITLE
Ignore Autogenerated column when updating

### DISCUSSION THREAD

**Discussion by joranboc** [slick/slick#2849] [Questions]
Hello,
We have this class to reuse in our models to update rows
![image](https://github.com/slick/slick/assets/6061867/9d8641a4-d232-4a72-8ded-da8070b32ecf)
and we created an autogenerated column with Mysql8 something like
`ALTER TABLE people ADD COLUMN is_tester TINYINT(1) GENERATED ALWAYS AS ( email is not null and email like '%+test%%') STORED after email;`
but when trying to update we're having this error, is there a way to filter the autogenerated column? 
![image](https://github.com/slick/slick/assets/6061867/d90881ce-29a6-4cf6-9cc3-536d3d6865ed)

Thanks for you help

**Comment by nafg** [comment]
https://github.com/nafg/slick-additions can help with that.

Or, you could take a similar approach: Add a type parameter to your trait that represents the model without the ID, an abstract method for the corresponding projection, and use those directly.

That said, I'm not sure if what your seeing is expected behavior or not, and if not if it's a bug in Slick or in your code.

Can you share a self-contained reproduction of the issue? (It could be a separate project, or a PR adding a test to Slick.)

Otherwise we could set up a call if you like.
